bluepysnap/spike_report.py

```diff
@@ -99,37 +99,53 @@ def get(self, group=None, t_start=None, t_stop=None):
             t_stop (float): Include only spikes occurring before this time.
 
         Returns:
-            pandas.Series/np.array:
-                If single node ID is passed as ``group`` returns a np.array of spiking times.
```

Comment:
I don't know who was using this function before, so I am just asking you to confirm that there is no code which passes a single node ID anymore.

Reply:
This is not used yet. So this is fine to change it now.